dlt/destinations/impl/bigquery/configuration.py

```diff
@@ -10,7 +10,7 @@
 
 @configspec
 class BigQueryClientConfiguration(DestinationClientDwhWithStagingConfiguration):
-    destination_name: str = "bigquery"
+    destination_type: Final[str] = "bigquery"  # type: ignore
```

Comment:
interesting option would be to have `destination_name = "bigquery"` as well. so this is more explicit. but up to you! maybe better to mark it as Final in base class and never redefine

Reply:
i think i prefer to have it the way it is now